.github/workflows/k8s-access.yaml

```diff
@@ -0,0 +1,41 @@
+on: push
+name: query
+jobs:
+  deploy:
+    name: query cluster
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.DEVOPS_BOT_AWS_ACCESS_KEY }}
+          aws-secret-access-key: ${{ secrets.DEVOPS_BOT_AWS_SECRET_ACCESS_KEY }}
+          aws-region: eu-west-1
+
+      - name: ASDF - Install
+        uses: asdf-vm/actions/setup@v1
+        with:
+          asdf_branch: v0.9.0
+
+      - name: ASDF - Install Tools
+        run: |
+          asdf plugin add awscli || true
+          asdf plugin add kubectl || true
+          asdf install awscli 2.7.18
+          asdf install kubectl 1.24.7
+
+      - name: Test authentication
+        run: |
+          aws eks update-kubeconfig --name k8s-kurated-addons
+          kubectl config set-credentials ci-bot --token=$K8S_GHA_TOKEN
+          kubectl config set-context k8s-kurated-addons-gha --cluster=arn:aws:eks:eu-west-1:$AWS_ACCOUNT_ID:cluster/k8s-kurated-addons --user=ci-bot
+          kubectl config use-context k8s-kurated-addons-gha
+          kubectl get namespaces
+          kubectl get pods
+
+        env:
+          K8S_GHA_TOKEN: ${{ secrets.DEVOPS_BOT_K8S_SERVICE_ACCOUNT_TOKEN }}
+          AWS_ACCOUNT_ID: ${{ secrets.DEVOPS_BOT_K8S_AWS_ACCOUNT_ID }}
```

Comment:
@blucas I don't think we will need this action in the repo, did you create this for testing the functionality?

Reply:
correct, just to prove out that it works.  Once all other feedback is addressed, I'll remove the workflow prior to merging.